templates/profiles.html

```diff
@@ -561,6 +611,20 @@ <h4 id="name" style="margin:5px; margin-top:5px;"></h4>
             keepOrder: true });
     }
 
+    function addAdditionalInfo(){
+        let $table_row = $('#additional-info-row-template-holder').find('#additional-info-template').clone();
+        $('#ability-additional-info').find('#additional-info-table').append($table_row);
+        $('#ability-additional-info').show();
+        $table_row.find('.ability-remove').click(function(){
+<!--    TODO: if deleting last row, hide additional info div    -->
```

Comment:
no TODO's.. if we need to do it, let's do it

Reply:
Ugh... I actually did implement it, just forgot to delete the TODO.